action/jsonapi.go

```diff
@@ -91,7 +92,7 @@ func (s *Action) getLibPath(ctx context.Context, c *cli.Context, browser string,
 func (s *Action) getWrapperPath(ctx context.Context, c *cli.Context) (path string, err error) {
 	path = c.String("path")
 	if path == "" {
-		path, err = s.askForString(ctx, color.BlueString("In which path should gopass_wrapper.sh be installed?"), manifest.DefaultWrapperPath)
+		path, err = s.askForString(ctx, color.BlueString("In which path should gopass_wrapper.sh be installed?"), config.Directory())
```

Comment:
I'm not sure if it's a good idea to place the script in the config folder, but as the user can still choose another location this should be OK.

Reply:
Yeah, I am also not fully happy about this, but as far as I know, there's not FHS compliant location to store user scripts that shouldn't be reachable via $PATH

Reply:
Yes, I agree.